Gateway/Request/APMBuilder.php

```diff
@@ -91,7 +92,7 @@ public function build(array $buildSubject)
             case Truemoney::CODE:
                 $paymentInfo[self::SOURCE] = [
                     self::SOURCE_TYPE      => 'truemoney',
-                    self::SOURCE_PHONE_NUM => $method->getOrder()->getBillingAddress()->getTelephone(),
+                    self::SOURCE_PHONE_NUM => $method->getAdditionalInformation(TruemoneyDataAssignObserver::PHONE_NUMBER),
```

Comment:
We don't want to replace `SOURCE_PHONE_NUM` to `SOURCE_PHONE_NUMBER` as well?

Reply:
i missed that.. :|